scripts/python/services/GithubService.py

```diff
@@ -27,6 +30,27 @@ def close_expired_issues(self, repository_name: str):
 
     def __is_expired(self, issue: Issue):
         grace_period = issue.created_at + timedelta(days=45)
-        return (issue.title == "User access removed, access is now via a team"
+        return (issue.title == self.USER_ACCESS_REMOVED_ISSUE_TITLE
                 and issue.state == "open"
                 and grace_period < datetime.now())
+
+    def create_an_access_removed_issue_for_user_in_repository(self, user_name: str, repository_name: str):
```

Comment:
A few things to note with the refactors:

Removed timeout - rate limiting functionality will be handled centrally after all functions have been migrated.
Removed exception handling - prevent us from failing silently